resinhup.py

```diff
@@ -47,6 +47,8 @@ def main():
                       help = "Configuration file to be used. Default: " + default_resinhup_conf_file)
     parser.add_argument('-f', '--force', action = 'store_true', dest = 'force', default = False,
```

Comment:
No. It is for all the other checks but version (currently the remaining ones are fingerprints and custom modules I think).